LAKMobile/src/screens/LoginScreen.tsx

```diff
@@ -19,7 +19,7 @@ export function LoginScreen({ navigation }: LoginProps) {
   const auth = useContext(AuthContext);
 
   const validatePhone = (): boolean => {
-    const phoneRegex = new RegExp('^[0-9]{10}$');
+    const phoneRegex = new RegExp('[0-9+]');
```

Comment:
Does this regex do what we want it to? it looks like it just checks for any digit or a + sign and marks it as valid